### PR TITLE
Disable CrossgenOutput for loongarch64 architecture

### PR DESCRIPTION
**Pull request by chenguohui**
Loongarch64 architecture is not support crossgen2 yet.

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
@wtgodbe can you please review this? Thanks!

File: src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj [hunk 1]
```diff
@@ -41,8 +41,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <!-- Pack .ni.r2rmap files in symbols package (native symbols for Linux) -->
     <AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>$(AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder);.r2rmap</AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>
 
-    <!-- Optimize the framework using the crossgen2 tool. Crossgen2 is not currently supported on s390x or ppc64le or armv6. -->
-    <CrossgenOutput Condition=" '$(TargetArchitecture)' == 's390x' OR '$(TargetArchitecture)' == 'armv6' OR '$(TargetArchitecture)' == 'ppc64le' ">false</CrossgenOutput>
+    <!-- Optimize the framework using the crossgen2 tool. Crossgen2 is not currently supported on s390x or ppc64le or armv6 or loongarch64. -->
+    <CrossgenOutput Condition=" '$(TargetArchitecture)' == 's390x' OR '$(TargetArchitecture)' == 'armv6' OR '$(TargetArchitecture)' == 'ppc64le' OR '$(TargetArchitecture)' == 'ppc64le' ">false</CrossgenOutput>
```
Comment (wtgodbe) [inline src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj:45]: Is this supposed to be `loongarch64`?

**Review by wtgodbe — CHANGES_REQUESTED** [review]
It looks like you have a typo, right now you're just duplicating the previous condition

File: src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj (hunk 1 above)
Comment (chenguohui) [reply src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj]: Yes. It should be @loongarch64@, sorry for my mistake.

**Comment by chenguohui** [comment]
> It looks like you have a typo, right now you're just duplicating the previous condition

@wtgodbe I have corrected the typo. And thank you for you patience.

**Review by wtgodbe — APPROVED** [review]

**Comment by wtgodbe** [comment]
Looks good, thanks for the change!

**Pull request merged**